CMakeLists.txt

```diff
@@ -24,41 +27,19 @@ endif("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
 # Set local module path.
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 
-project(FreeDV)
-
 # Build universal ARM64 and x86_64 binaries on Mac.
 if(BUILD_OSX_UNIVERSAL)
 set(CMAKE_OSX_ARCHITECTURES "x86_64;arm64")
 endif(BUILD_OSX_UNIVERSAL)
-
-# https://cmake.org/cmake/help/git-stage/policy/CMP0075.html
-if(POLICY CMP0075)
-    cmake_policy(SET CMP0075 NEW)
-endif()
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.11" CACHE STRING "Minimum OS X deployment version")
 
 # Return the date (yyyy-mm-dd)
-macro(DATE RESULT)
-  execute_process(COMMAND "date" "+%Y%m%d" OUTPUT_VARIABLE ${RESULT})
-endmacro()
-DATE(DATE_RESULT)
-string(REGEX REPLACE "\n$" "" DATE_RESULT "${DATE_RESULT}")
+string(TIMESTAMP DATE_RESULT "%Y-%m-%d" UTC)
 message(STATUS "Compilation date = XX${DATE_RESULT}XX")
 
 #
-# Set FreeDV version and generate src/version.h
+# generate src/version.h
 #
-set(FREEDV_VERSION_MAJOR 1)
-set(FREEDV_VERSION_MINOR 8)
-set(FREEDV_VERSION_PATCH 2)
-set(FREEDV_VERSION_SUFFIX "")
```

Comment:
How would suffix be handled after the changes? This tends to be used for test builds to make it easy to differentiate from official ones.

Reply:
Hmm, reading more, `*_VERSION_TWEAK` only accepts integer values, so changing the top of the file to "1.8.2-devel" or "1.8.2.devel" won't exactly work. I pushed a commit that brings that back as a separate CMake variable.